jaxlib/README.md

```diff
@@ -0,0 +1,5 @@
+# jaxlib: support library for JAX
+
+jaxlib is the support library for JAX, and packages all the compiled dependencies
```

Comment:
I wonder if we have a better description of jaxlib somewhere

Reply:
I didn't find anything, but this seems like it would be the canonical place for it 😁

Reply:
Updated based on an existing resource from @hawkinsp